gc_tests/tests/alloc_info_oom.rs

```diff
@@ -0,0 +1,13 @@
+// Run-time:
+//  status: error
```

Comment:
It might not be a bad idea to test for specific `stderr` output here? Even something simple like:

```
stderr: ...out of memory...
```

just so that other types of errors don't get confused for the one you're intending to test.

Reply:
Good point. I'll add this.